cli/cli/command_framework/lowlevel/lowlevel_kurtosis_command.go

```diff
@@ -371,3 +374,24 @@ func renderArgUsageStr(arg *args.ArgConfig) string {
 	}
 	return result
 }
+
+// this method removes the file path from the error
+//TODO: this is currently not integrated but will do once all the necessary PRs are merged.
+func removeFilePathFromErrorMessage(errorMessage string) error {
+	errorMessageConvertedInList := strings.Split(errorMessage, separator)
+
+	// only the even numbered elements needs to be picked.
+	var cleanErrorList []string
+	for i, arr := range errorMessageConvertedInList {
+		if i%2 == 0 {
+			cleanErrorList = append(cleanErrorList, strings.Trim(arr, "\n"))
+		}
+	}
+
+	if logrus.GetLevel() == logrus.DebugLevel {
+		logrus.Debug(errors.New(errorMessage))
+	} else {
+		out.FileLogger.Error(errors.New(errorMessage))
+	}
```

Comment:
this part seems to be `out of the method scope` and brakes the single responsibility principle for functions. I suggest you move it to another method and probably call it before calling this one.

Reply:
good point; I can do this in the next PR when i am actually integrating the  method